README.md

```diff
@@ -5,20 +5,35 @@
 
 
 ## About
-- Ansible Network Backup Collection contains the role which provides a platform-agnostic way of managing network backup on supported network platforms. This collection provides the user the capabilities to create, compare and tag backups supporting local and remote data stores.
+- Ansible Network Backup Collection contains the role `backup` which provides a platform-agnostic way of managing network backup on supported network platforms. This collection provides the user the capabilities to create, compare and tag backups supporting local and remote data stores.
```

Comment:
Nitpick: Reading the first line gives me the impression that this VC _only_ supports backup. Can we have a introductory line on the two functionalities that this VC provides and then expand on each of them? This way it'd appear more cohesive.

Secondly, are we actually okay calling this a `collection` and not `validated content`?